app/src/main/res/values/strings.xml

```diff
@@ -700,4 +700,5 @@
     <string name="recent">Recent</string>
     <string name="chapters">Chapters</string>
     <string name="no_app_to_open_intent">No app on your device can open this</string>
+    <string name="download_has_started">Download has started</string>
```

Comment:
Maybe it should be `Downloading $VIDEO_NAME` instead.

Reply:
I don't see it as needed. The issue does not ask for this.

Reply:
> @WoodyMats thanks for opening a PR! Could you please not remove the template next time you open a PR?

Sorry! Now i just see the PR template. In future PRs i will do it with the right way.

Reply:
Can you explain me the issue so i can do it with the right way? :)